### DISCUSSION TITLE
Additional docs for the admin customization

### DISCUSSION THREAD

**Discussion by filipesmedeiros** [strapi/strapi#6155] [Legacy Topics]
Hey guys, loving Strapi!

I just changed the 'true'/'false' strings in the list for emojis (UTF8 chars), specifically a green check mark and a red cross. It's obviously nothing much, but do you guys want me to add it to the docs as an example, because it messes with the React component part, and I think the docs only give an example about a utils function.

Also, this seems like something a lot of people would like, right?

**Comment by lauriejim** [comment]
Hello! I'm not sure to understand what you are talking about.
Can you please precise please.

**Reply by filipesmedeiros** [reply]
Yes, sorry.

The list of entries of a type shows a set of attributes of each entry. If the attribute is a Boolean, the list shows "true" or "false". I think this is a good neutral default, but many times an icon/emoji of a cross and checkmark will provide a better user experience for people who manage content.

I suggested adding a small section in the guides showing this customization, for people who might want it but don't want to go through the trouble of figuring out how to do it on their own (people less experienced in react or development in general).

**Reply by lauriejim** [reply]
Ho okay, got it, I will create a Jim's recipes video for that - [playlist](https://www.youtube.com/playlist?list=PL7Q0DQYATmvhlHxHqfKHsr-zFls2mIVTi)
